app/views/project_features/index.xlsx.axlsx

```diff
@@ -0,0 +1,37 @@
+wb = xlsx_package.workbook
+
+s = wb.styles
+centered_bold_15 = s.add_style alignment: { horizontal: :center }, b: true, sz: 15
+centered_bold = s.add_style alignment: { horizontal: :center }, b: true,
+                            border: { style: :medium, color: '000000' }
+vertical_top_bold  = s.add_style alignment: {vertical: :top}, b: true
+centered = s.add_style alignment: { horizontal: :center, wrap_text: true }, border_right: { style: :medium, color: '000000' }, border_left: { style: :medium, color: '000000' }
+border_cell_blr_bold = s.add_style border_right: { style: :medium, color: '000000' }, border_bottom: { style: :medium, color: '000000' }, border_left: { style: :medium, color: '000000' }, b: true,
+                                   alignment: { horizontal: :center }
+
+wb.add_worksheet(name: "Features Worksheet") do |sheet|
+  month_year_row = params[:month_year]
+  month_year_row ||= "All"
+  sheet.add_row [month_year_row], style: centered_bold_15
+  sheet.add_row [""]
+  sheet.merge_cells("A1:G1")
+  project_feature_index = 3
+  @project_features.each do |project_feature|
+    sheet.add_row ["Project name: #{project_feature.project_name}\nProject id: #{project_feature.project_id}"], style: vertical_top_bold
+    sheet.add_row [""]
+    sheet.merge_cells("A#{project_feature_index}:G#{project_feature_index+1}")
+    sheet.add_row ["Name", "Description", "Waste description", "Effort saved", "Repeat time", "Repeat unit", "Man month"], style: centered_bold
+    @project = Project.find_by id: project_feature.project_id
+    total_man_month = 0
+    @project&.project_features&.filter_name(params[:name]).filter_month(@month).filter_year(@year).each do |feature|
```

Comment:
c thấy sao sao đó nhỉ? sao k filter ở ngoài sau đó export ra all project feature thỏa mãn mà vào trong lại còn phải lặp

Reply:
![image](https://github.com/awesome-academy/project_management/assets/142207375/ab8170c4-efcd-4c5c-9c66-a3d2f60b497c)
ở ngoài đang đang xử lý theo kiểu lấy project_feature rồi group lại ạ c, k lấy được toàn bộ các trường ạ

Reply:
để em sửa lạy logic chỗ này ạ, em thấy cũng hơi cấn